packages/core/src/types-hoist/options.ts

```diff
@@ -10,6 +10,11 @@ import type { StackLineParser, StackParser } from './stacktrace';
 import type { TracePropagationTargets } from './tracing';
 import type { BaseTransportOptions, Transport } from './transport';
 
+interface IgnoreSpanFilter {
```

Comment:
m: I think we can type this a bit smarter, right? Currently, this allows users to pass in an empty object but we could type it in a way so that at least one prop is required 
```suggestion
type IgnoreSpanFilter = {
  name: string | RegExp;
  op?: string | RegExp;
} | {
  name?: string | RegExp;
  op: string | RegExp;
}
```

we should probably still check that this is also enforced when we apply the option (which we do already)

Reply:
Udpated the type to this